frontend/src/views/funding/FundingOverviewView.vue

```diff
@@ -54,47 +163,211 @@
   data() {
     return {
+      format,
       tab: null,
-      fundingAgreements: [
-        {
-          guid: '21d0d405-900c-ef11-9f89-000d3af44283',
-          faNumber: 'FA-123456',
-          faType: 'Base Funding',
-          facility: 'Facility 1',
-          startDate: 'Nov 22, 2023',
-          endDate: 'Nov 22, 2024',
-          status: 'FA Signature Pending',
-        },
-        {
-          guid: '21d0d405-900c-ef11-9f89-000d3af44283',
-          faNumber: 'FA-000002',
-          faType: 'Supplementary Allowances',
-          facility: 'Facility 2',
-          startDate: 'May 15, 2023',
-          endDate: 'May 15, 2024',
-          status: 'FA Submitted to Ministry',
-        },
-      ],
+      loading: false,
+      fundingAgreements: [],
+      orgUsers: [],
+      selectedFacility: null,
+      selectedDateFilterType: null,
+      selectedPaymentFilterTypes: [],
+      startDateFrom: null,
+      startDateTo: null,
+      menuStartDateFrom: false,
+      menuStartDateTo: false,
       headers: [
-        { title: 'FA Number', key: 'faNumber' },
-        { title: 'FA Type', key: 'faType' },
-        { title: 'Facility', key: 'facility' },
+        { title: 'Funding Agreement Number', key: 'fundingAgreementNumber' },
+        { title: 'Funding Agreement Type', key: 'fundingAgreementType' },
+        { title: 'Facility Name', key: 'facility' },
+        { title: 'Signing Expense Authority', key: 'expenseAuthorityName' },
         { title: 'Start Date', key: 'startDate' },
         { title: 'End Date', key: 'endDate' },
         { title: 'Status', key: 'status' },
         { title: 'Actions', key: 'actions' },
       ],
+      dateFilterTypes: [
+        { label: '3 Months', value: '3Months' },
+        { label: '6 Months', value: '6Months' },
+        { label: 'YTD', value: 'YTD' },
+        { label: 'Custom', value: 'Custom' },
+      ],
+      paymentFilterTypes: [
+        { label: 'Base Funding', value: 'baseFunding' },
+        { label: 'Supplementary Allowances', value: 'supplementaryAllowances' },
+        { label: 'Other', value: 'other' },
+      ],
+    }
+  },
+  computed: {
+    ...mapState(useAuthStore, ['userInfo']),
+    formattedStartDateFrom() {
+      return this.startDateFrom ? format.formatDate(this.startDateFrom) : ''
+    },
+    formattedStartDateTo() {
+      return this.startDateTo ? format.formatDate(this.startDateTo) : ''
     }
   },
+  async created() {
+    this.FUNDING_AGREEMENT_STATUS_CODES = FUNDING_AGREEMENT_STATUS_CODES
+    this.initialize()
+    await this.getOrgUsers() // Retrieve the list of users to retrieve the expense authority name
+    await this.getFundingAgreements()
+  },
+  methods: {
+    initialize() {
+      this.selectedFacility = null
+      this.selectedDateFilterType = '3Months'
+      this.selectedPaymentFilterTypes = ['baseFunding', 'supplementaryAllowances', 'other']
+      this.startDateFrom = null
+      this.startDateTo = null
+    },
+
+    async getOrgUsers() {
```

Comment:
I prefer "load" for methods that aren't just getters or http GET calls. Same applies to `getFundingAgreements()`.

Reply:
Updated.

Reply:
I wasn't clear enough here. You could keep loadFundingAgreements() and loadOrgUsers() separate if you prefer...just wanted getFoo changed to loadFoo. Given orgUsers is just one line I would keep it code structured as is but rename load() to loadFundingAgreements().

Reply:
Fixed.